Epam.TestAutomation/Epam.TestAutomation.API/Controllers/PhoneController.cs

```diff
@@ -50,5 +50,15 @@ public PhoneController(CustomRestClient client) : base(client, Service.Phones)
         {
             return Post<T, PhoneModel>(PhonesResource, model);
         }
+
+        public (RestResponse Response, T Phone) DeletePhone<T>(string phoneId)
```

Comment:
please add summaries. Otherwise it would be impossible to fins which model should be used later!

Reply:
SUMMARIES!